ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -57634,7 +57634,7 @@ periodics:
     repo: release
   interval: 12h
   labels:
-    ci.openshift.io/cluster: build01
+    ci.openshift.io/build-farm: build01
```

Comment:
Do we need to change this file?   This is an  openshift/release level master file and I don't see any references to openshift-qe.   But I might not understand this :-)

Reply:
After editing my 2 files it told me I needed to run make builds/job and those updates were from that

Reply:
On master I see it as build-farm.   I think git is not recognizing it as a merge conflict because it is generated.   I think it is safe to drop any commits for those files.   But who know what that might break?    Or put your changes on top of a fresh branch of master.